src/test/java/com/wikia/webdriver/testcases/renametool/RenameToolTests.java

```diff
@@ -78,13 +78,12 @@ public void UserAlreadyRenamedMessageShowed() {
 
   @Test
   @Execute(asUser = User.QARENAME)
-  @RelatedIssue(issueID = "SUS-123", comment = "Ten test nie działą")
   public void GoToHelpPage() {
     SpecialRenameUserPage renameUserPage = new SpecialRenameUserPage()
         .open();
     HelpPage helpPage = renameUserPage.goToHelpPage();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "helpPage" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1481)